README.md

```diff
@@ -30,14 +30,14 @@ import toopher
 api = toopher.ToopherApi("<your consumer key>", "<your consumer secret>")
 
 # Step 1 - Pair with their phone's Toopher app
-pairing_status = api.pair("pairing phrase", "username@yourservice.com")
+pairing = api.pair("pairing phrase", "username@yourservice.com")
 
```

Comment:
We need to check that the user has responded affirmatively like we do after the `authenticate` call below--can you please add a similar check here? Note that pairings are `enabled` rather than `granted`.